src/UnitfulAstro.jl

```diff
@@ -86,9 +86,76 @@ import UnitfulAngles: arcminute, arcsecond
 # Total electron content unit (used for ionospheric physics and low-frequency radio astronomy)
 @unit TECU       "TECU"     TotalElectronContentUnit  1e16*m^-2                 false
 
+@logscale mag "mag" Magnitude 10 -2.5 false
+
+#https://en.wikipedia.org/wiki/AB_magnitude     
+@logunit    AB_mag  "AB mag"        Magnitude   3631Jy
+
+# Bolometric magnitudes (cf IAU 2015)
+# https://en.wikipedia.org/wiki/Absolute_magnitude#Bolometric_magnitude
+@logunit    bol_Mag "bol Mag"       Magnitude   3.0128e28*W
+@logunit    bol_mag "bol mag"       Magnitude   2.518_021_002e-8*W*m^-2
+
+#https://en.wikipedia.org/wiki/Apparent_magnitude#Standard_reference_values
+#https://www.cfa.harvard.edu/~dfabricant/huchra/ay145/mags.html
+@logunit    U_mag   "Johnson U mag" Magnitude   1810Jy
+@logunit    B_mag   "Johnson B mag" Magnitude   4260Jy
+@logunit    V_mag   "Johnson V mag" Magnitude   3640Jy
+@logunit    R_mag   "Johnson R mag" Magnitude   3080Jy
+@logunit    I_mag   "Johnson I mag" Magnitude   2550Jy
+@logunit    J_mag   "Johnson J mag" Magnitude   1600Jy
+@logunit    H_mag   "Johnson H mag" Magnitude   1080Jy
+@logunit    K_mag   "Johnson K mag" Magnitude    670Jy
+@logunit    g_mag   "Gunn g mag"    Magnitude   3730Jy
+@logunit    r_mag   "Gunn r mag"    Magnitude   4490Jy
+@logunit    i_mag   "Gunn i mag"    Magnitude   4760Jy
+@logunit    z_mag   "Gunn z mag"    Magnitude   4810Jy
+
+Unitful.isrootpower_dim(::typeof(Unitful.dimension(AB_mag))) = false
```

Comment:
I'm pretty sure that also this is type piracy and shouldn't be here.

Reply:
This is how [Unitful.jl handles units](https://github.com/ajkeller34/Unitful.jl/blob/master/src/pkgdefaults.jl#L225)
```jl
# TODO: some more dimensions?
isrootpower_dim(::typeof(dimension(W)))         = false
isrootpower_dim(::typeof(dimension(V)))         = true
isrootpower_dim(::typeof(dimension(A)))         = true
...
```

Reply:
Also in this case, `isrrotpower_dim` is a function defined only in that package (unless I'm missing something) and `dimension(AB_mag)` I'm pretty sure is a generic `Unitful` dimension that someone else may be using in their code.

Reply:
`dimension(AB_mag)` is defined in this PR not in `Unitful.jl`

Reply:
Precompiling the package gives this warning:
```
WARNING: Method definition isrootpower_dim(Unitful.Dimensions{(Unitful.Dimension{:Mass}(power=Base.Rational{Int64}(num=1, den=1)), Unitful.Dimension{:Time}(power=Base.Rational{Int64}(num=-2, den=1)))}) in module Unitful at /home/mose/.julia/packages/Unitful/W0mMi/src/pkgdefaults.jl:230 overwritten in module UnitfulAstro at /home/mose/.julia/dev/UnitfulAstro/src/UnitfulAstro.jl:114.
```
Doesn't sound quite right.

> `dimension(AB_mag)` is defined in this PR

No, `AB_mag` is defined in this PR, not its dimension:

```julia
julia> dump(Unitful.dimension(UnitfulAstro.AB_mag))
Unitful.Dimensions{(Unitful.Dimension{:Mass}(1//1), Unitful.Dimension{:Time}(-2//1))} 𝐌 𝐓^-2
```

This is by no means a dimension unique to this package.

Reply:
Actually looks like this line isn't need.

Reply:
You see? :wink: